README.md

```diff
@@ -8,7 +8,7 @@ variables, functions and methods. Within each of these categories, exported decl
 ## Installation
 
 ```sh
-$ go get github.com/jdeflander/goarrange
+$ go install github.com/jdeflander/goarrange
```

Comment:
Will this also work for Go 1.13?

Reply:
I have an M1-chip (arm64), so I can not test older versions, such as v1.13. (afaik)